src/main/java/io/github/mivek/parser/AbstractParser.java

```diff
@@ -133,7 +121,7 @@ protected void parseDeliveryTime(final AbstractWeatherCode pWeatherCode, final S
      * @param pPart      the token to parse.
      * @return boolean if the token pPart as been parsed.
      */
-    protected boolean generalParse(final AbstractWeatherContainer pContainer, final String pPart) {
+    boolean generalParse(final AbstractWeatherContainer pContainer, final String pPart) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/mivek/MetarParser/pullRequest?prid=4593741)